themis/languages/objc/installation.md

```diff
@@ -59,6 +48,10 @@ Themis is [available via **CocoaPods**](https://cocoapods.org/pods/themis).
 
 #### Using BoringSSL
 
+{{< hint warning >}}
+BoringSSL subspec is deprecated and will be removed in Themis 0.14. Please switch to the default subspec.
```

Comment:
```suggestion
`themis-boringssl` subspec is deprecated and will be removed in Themis 0.14. Please switch to the default subspec.
```

Reply:
But the default subspec is `opessl-1.1.1` and it is better not to specify it at all.... hmmm
I think it would be better to say: 
Please switch to the default subspec by using `pod 'themis'` in your Podfile.

Reply:
```suggestion
`themis-boringssl` subspec is deprecated and will be removed in Themis 0.14. Please switch to the default subspec by using pod 'themis' in your Podfile.
```

Reply:
yes, you are right, let's use our wording